app/controllers/users_controller.rb

```diff
@@ -9,13 +9,14 @@ def new
   end
 
   def index
+    @users = User.includes(:associate_consultant,
+                          {associate_consultant: :reviewing_group},
```

Comment:
Align the parameters of a method call if they span more than one line.<br>Space inside { missing.<br>Space inside } missing.